src/zeit/content/article/edit/volume.py

```diff
@@ -0,0 +1,47 @@
+from zeit.cms.i18n import MessageFactory as _
+import grokcore.component as grok
+import zeit.cms.content.reference
+import zeit.content.article.edit.interfaces
+import zeit.content.article.edit.reference
+import zeit.content.volume.interfaces
+
+
+class ReferenceProperty(zeit.cms.content.reference.SingleReferenceProperty):
```

Comment:
Für Artikelmodule, die einen anderen Content referenzieren _und sonst weiter nix_ (was bei Volume der Fall ist), gibt's die Mechanik schon komplett fertig in z.c.article.edit.reference. Da kann man einfach eine weitere Subklasse anfügen, und nur den Contenttyp anpassen (vgl Infobox, Gallery). Diese ganze Datei hier ist dann unnötig.